deeprank/generate/DataGenerator.py

```diff
@@ -577,7 +579,7 @@ def add_feature(self, remove_error=True, prog_bar=True):
                     aug_molgrp.create_dataset("features/" + k, data=data)
 
                     # rotate
-                    self._rotate_feature(
+                    DataGenerator._rotate_feature(
```

Comment:
same here, why DataGenerator. and not self._rotate_feature ?

Reply:
Updated now.